torchao/dtypes/affine_quantized_tensor.py

```diff
@@ -663,6 +700,99 @@ def get_plain(self) -> Tuple[torch.Tensor, torch.Tensor, torch.Tensor]:
     def get_layout_type(self) -> LayoutType:
         return self.layout_type
 
+
+@register_layout_cls(FpxLayoutType)
+class FpxAQTLayout(AQTLayout):
```

Comment:
this is added in the fpx PR..do you want to wait until that is landed?

Reply:
As discussed, I'm not using this layout anymore. I'll remove this code.